SPECS/kernel/config

```diff
@@ -1014,7 +1014,7 @@ CONFIG_COMPAT_BINFMT_ELF=y
 CONFIG_ELFCORE=y
 CONFIG_CORE_DUMP_DEFAULT_ELF_HEADERS=y
 CONFIG_BINFMT_SCRIPT=y
-CONFIG_BINFMT_MISC=m
+CONFIG_BINFMT_MISC=y
```

Comment:
we explicitly made module here: https://github.com/microsoft/azurelinux/commit/2e790881e42f4b8e97d8ecf91ad4379064dbd069
check to see why